### PR TITLE
Fix for issue#27 :  Integrating the security validation mechanism for apiKeyAuth and basicAuth validations to OamLog

### PR DESCRIPTION
**Pull request by vanithavalluripalli9**

### CODE REVIEW

File: server/controllers/Core.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ var oamLogService = require('onf-core-model-ap-bs/basicServices/OamLogService');
 
 module.exports.getControlConstruct =async function getControlConstruct (req, res, next) {
```
Comment (PrathibaJee) [inline server/controllers/Core.js:9]: Remove the un used authorization import

File: server/controllers/IntegerProfile.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ var oamLogService = require('onf-core-model-ap-bs/basicServices/OamLogService');
```
Comment (PrathibaJee) [inline server/controllers/IntegerProfile.js:8]: Remove the un used authorization import

File: server/controllers/OamRecordProfile.js [hunk 3]
```diff
@@ -8,7 +8,6 @@ var oamLogService = require('onf-core-model-ap-bs/basicServices/OamLogService');
```
Comment (PrathibaJee) [inline server/controllers/OamRecordProfile.js:8]: Remove the un used authorization import

File: server/controllers/OperationClient.js [hunk 4]
```diff
@@ -8,7 +8,6 @@ var oamLogService = require('onf-core-model-ap-bs/basicServices/OamLogService');
```
Comment (PrathibaJee) [inline server/controllers/OperationClient.js:8]: Remove the un used authorization import

File: server/controllers/OperationServer.js [hunk 5]
```diff
@@ -8,7 +8,6 @@ var oamLogService = require('onf-core-model-ap-bs/basicServices/OamLogService');
```
Comment (PrathibaJee) [inline server/controllers/OperationServer.js:8]: Remove the un used authorization import

File: server/controllers/TcpServer.js [hunk 6]
```diff
@@ -8,7 +8,6 @@ var oamLogService = require('onf-core-model-ap-bs/basicServices/OamLogService');
```
Comment (PrathibaJee) [inline server/controllers/TcpServer.js:8]: Remove the un used authorization import

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Please Remove the un used authorization import.

**Review by vanithavalluripalli9 — COMMENTED** [review]
Removed the un used authorization import

**Pull request merged**